jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/deadlines/notifications/impl/email/EmailNotificationListener.java

```diff
@@ -60,19 +60,14 @@
     
     private static final Logger logger = LoggerFactory.getLogger(EmailNotificationListener.class);
 
-    private UserInfo userInfo;
-
     private Session mailSession = EmailSessionProducer.produceSession();
     
-    public EmailNotificationListener(UserInfo userInfo) {
-    	this.userInfo = userInfo;
-    }
     
     @Override
-    public void onNotification(NotificationEvent event) {
-
+    public void onNotification(NotificationEvent event, Object... params) {
+        UserInfo userInfo = (UserInfo) params[0];
```

Comment:
main reason I used varargs instead of direct UserInfo argument is that most likely we will have more parameters needed when providing custom types of notifications and I'd like to avoid changing the NotificationListener interface every time new param is added.
Though when more votes would be to keep it strict typed I will change it :) thanks @manstis